components/Layout/LeadSpace/LayoutLeadSpaceWithCard.vue

```diff
@@ -65,7 +65,7 @@ defineProps<Props>();
     padding-top: carbon.$spacing-09;
   }
 
-  &__main {
+  &__div {
```

Comment:
depending on what class name you choose, you'll need to replace this selector:

- `&__hero` or
- `&__content`